pkg/network/cache/fake/infocache.go

```diff
@@ -99,3 +115,24 @@ func (f *fakeDomainInterfaceStore) Write(ifaceName string, cacheInterface *api.I
 	f.store[ifaceName] = cacheInterface
 	return nil
 }
+
+type fakeVifCacheStore struct {
+	lock  *sync.Mutex
```

Comment:
We can use here a https://golang.org/pkg/sync/#RWMutex, so if multiple goroutintes do a read they don't lock each other.

Reply:
I'm extending what we have - this is how it's done for the other 2 caches.

I think that comment belonged on the original PR that introduced the caching scheme.

It can always be done in a follow up PR to all the fake caches.